### ISSUE TITLE
`yarn start` error: Module not found: Can't resolve '@ts-ast-viewer/shared'

### ISSUE THREAD

**Issue by hantatsang** [dsherret/ts-ast-viewer#86]
I'm having trouble running the site locally.

After `yarn install`, I run `yarn start`:

- `npm run updateCompilerFIles`  completes successfully
- `npm run start --prefix site` shows _"Starting the development server"_, then this error:

```
Failed to compile.

./src/AppContext.tsx
Module not found: Can't resolve '@ts-ast-viewer/shared' in 'C:\Users\user\ts-ast-viewer\site\src'
```

![Screenshot 2021-06-16 215410](https://user-images.githubusercontent.com/11912225/122214792-d9b4ae00-ceed-11eb-8ea9-593a16da929e.png)

Am I missing something in the steps to run the site?

### System Info

- OS: Windows 10
- Node: v14.16.0
- Yarn: v1.22.10
- npm: 6.14.11

**Comment by bsafwen** [comment]
same here

**Comment by dsherret** [comment]
Sorry, I made some changes and then forgot to update the setup instructions. I've added a `yarn setup` command. Run that before `yarn start` and you should be good.

Fixed in 078bb66
